### PR TITLE
Docs/add contribution templates

### PR DESCRIPTION
**Pull request by jelledouwe**
Add templates for contributing, for creating issues etc.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve
+labels: bug
+title: "[Bug] bug title"
+---
+
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.
```
Comment (bheijden) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: Dit misschien weg halen. Is wel misschien wel erg copy paste haha

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F4DA Documentation"
+about: Report an issue related to EAGERx documentation
+labels: documentation
+---
+
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.
```
Comment (bheijden) [inline .github/ISSUE_TEMPLATE/documentation.md:8]: same hier

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+name: "\U0001F680Feature Request"
+about: How to create an issue for requesting a feature
+labels: enhancement
+title: "[Feature Request] request title"
+---
+
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.
```
Comment (bheijden) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: same

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+name: ❓Question
+about: How to ask a question
+labels: question
+title: "[Question] question title"
+---
+
+**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
+Please post your question on the [RL Discord](https://discord.com/invite/xhfNqQv), [Reddit](https://www.reddit.com/r/reinforcementlearning/) or [Stack Overflow](https://stackoverflow.com/) in that case.
```
Comment (bheijden) [inline .github/ISSUE_TEMPLATE/question.md:9]: same

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have read the [CONTRIBUTION](https://github.com/eager-dev/eagerx/blob/master/CONTRIBUTING.rst) guide (**required**)
+- [ ] I have used semantic commit messages such that my PR is [semantic](https://github.com/zeke/semantic-pull-requests) (**required**)
+- [ ] I have updated the changelog accordingly (**required**).
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
+- [ ] I have updated the documentation accordingly.
+- [ ] I have reformatted the code using `make codestyle` (**required**)
+- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
```
Comment (bheijden) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: Maybe mention that we use black (+ where they can find the version)?

File: CONTRIBUTING.rst [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+      :return: describe what is returned
+      """
+      ...
+      return my_variable
+
+Pull Request (PR)
+#################
+
+Before proposing a PR, please open an issue, where the feature will be discussed.
+This prevent from duplicated PR to be proposed and also ease the code review process.
```
Comment (bheijden) [inline CONTRIBUTING.rst:82]: strange sentence

File: CONTRIBUTING.rst [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+Each PR need to be reviewed and accepted by at least one of the maintainers (@bheijden, @jelledouwe).
```
Comment (bheijden) [inline CONTRIBUTING.rst:84]: needs?

**Review by bheijden — CHANGES_REQUESTED** [review]
Nice, only a few comments

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (jelledouwe) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: In principle you are using the correct version when the Poetry environment is activated right?

Comment (bheijden) [reply .github/PULL_REQUEST_TEMPLATE.md:29]: Ah, true.

**Review by bheijden — APPROVED** [review]

**Pull request merged**